### PR TITLE
TW-986 HTTP clear response

### PR DESCRIPTION
**Pull request by jkonrath-postman**
This is the new feature content. It is an un-revert of a merge that went too early and was reverted.

This reverts commit c343e293415b82cec15d0b4eaced16c6c259743f, reversing changes made to 0921b326295fedcebcdf6aa84612523112843b82.

### CODE REVIEW

**Comment by jkonrath-postman** [comment]
I can't review this because I originally did the revert, but LGTM.

File: src/pages/docs/sending-requests/responses.md [hunk 1]
```diff
@@ -149,12 +149,13 @@ Postman displays the approximate size of the response. Hover over the response s
...
 
-* Select **Save as example** to save the response as an [example](/docs/sending-requests/examples/) that you can access later.
-* Select **Save to a file** to save the response as a JSON file.
+* Select **Save as Example** to save the response as an [example](/docs/sending-requests/examples/) that you can access later.
+* Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> then **Save response to file** to save the response as a JSON file.
+* Select the more actions icon <img alt="More actions icon" src="https://assets.postman.com/postman-docs/icon-more-actions-v9.jpg#icon" width="16px"> then  **Clear response** to remove any data in the response viewer. Note that for event-based requests, this is only available after the stream is closed.
 
-<img alt="Save an API response as an example or file" src="https://assets.postman.com/postman-docs/save-response.jpg" width="300px"/>
+<img alt="Save an API response as an example or file" src="https://assets.postman.com/postman-docs/v10/save-response-v10.jpg" width="300px"/>
```
Comment (mark-anthony-postman) [inline src/pages/docs/sending-requests/responses.md:158]:
```suggestion
<img alt="Save an API response as an example or file" src="https://assets.postman.com/postman-docs/v10/save-response-v10.jpg" width="505px"/>
```

Comment (mark-anthony-postman) [reply src/pages/docs/sending-requests/responses.md:158]: Image is 1010 pixels wide, so 505 px would be a better width

**Review by mark-anthony-postman — APPROVED** [review]
LGTM!

**Pull request merged**